src/components/Postmodal/PostModal.jsx

```diff
@@ -15,11 +24,17 @@ export const PostModal = () => {
           <VscChromeClose />
         </button>
         <div className="w-16 h-16 flex-shrink-0 mt-5">
-          <img
-            className="profile-avatar"
-            src="https://treatiseui.netlify.app/Images/ian-dooley-lg.jpg"
-            alt="profile dp"
-          />
+          {currentUser?.profileImg === "" ? (
```

Comment:
This dynamic avatar code is repeating multiple times. We can make a component out of it & pass `currentUser` & imageLink as props.